image-capture/grab-frame-take-photo.js

```diff
@@ -32,11 +32,15 @@ function onTakePhotoButtonClick() {
 
 /* Utils */
 
-function drawCanvas(canvas, imageBitmap) {
-  canvas.width = imageBitmap.width;
-  canvas.height = imageBitmap.height;
-  canvas.getContext('2d').drawImage(imageBitmap, 0, 0,
-      imageBitmap.width, imageBitmap.height);
+function drawCanvas(canvas, img) {
```

Comment:
It is called every time you press "Take Photo" buttron and "Grab Frame" button. See https://beaufortfrancois.github.io/samples/image-capture/grab-frame-take-photo.html